src/hotspot/share/services/management.cpp

```diff
@@ -736,9 +736,11 @@ JVM_ENTRY(jobject, jmm_GetMemoryUsage(JNIEnv* env, jboolean heap))
   MemoryUsage usage;
 
   if (heap) {
+    // HeapUsage = GC heap
     usage = Universe::heap()->memory_usage();
   } else {
     // Calculate the memory usage by summing up the pools.
+    // NonHeapUsage = CodeHeaps + Metaspace
```

Comment:
I think the new comments in this file can be removed.

Reply:
Thank you @iklam for review! All your comments have been addressed. I also fixed another jtreg failure(LowMemoryTest2.sh)